tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -144,3 +146,12 @@ def tap_import_certs(self):
         Wait(self.marionette).until(expected.element_displayed(element))
         element.tap()
         Wait(self.marionette).until(expected.element_displayed(*self._select_certs_screen_locator))
+    
+    def tap_active_wifi(self):
+        element = self.marionette.find_element(*self._select_active_wifi_locator)
+        element.tap()
```

Comment:
After here, it might be a good idea for a wait call for the element with #wifi-status to be displayed() and here.